goscraper.go

```diff
@@ -66,6 +128,23 @@ func (scraper *Scraper) Scrape() (*Document, error) {
 	return doc, nil
 }
 
+func (scraper *Scraper) GetDocument() (*Document, error) {
+	doc, err := scraper.getDocument()
```

Comment:
is it required to have this error dance in here or we can just `return scraper.getDocument()`?